plugins/channel_header_plug/channel_header_plug.test.tsx

```diff
@@ -81,6 +83,7 @@ describe('plugins/ChannelHeaderPlug', () => {
                 channel={{} as Channel}
                 channelMember={{} as ChannelMembership}
                 theme={{} as Theme}
+                sidebarOpen={false}
```

Comment:
Does it make sense add any test with `sidebarOpen={true}`?

Reply:
N/A since the variable was moved out of the component's state